cmd/neofs-cli/modules/container/set_eacl.go

```diff
@@ -147,4 +147,5 @@ func initContainerSetEACLCmd() {
 		"Increases default execution timeout to %.0fs", awaitTimeout.Seconds())) // simple %s notation prints 1m0s https://github.com/golang/go/issues/39064
 	flags.BoolP(commonflags.ForceFlag, commonflags.ForceFlagShorthand, false, "skip validation checks (ownership, extensibility of the container ACL)")
 	flags.BoolVar(&flagVarsSetEACL.noPreCheck, "no-precheck", false, "do not pre-check the extensibility of the container ACL")
+	_ = flags.MarkDeprecated("no-precheck", "use --force flag instead for skipping validation checks")
```

Comment:
We need an issue to drop it in some subsequent release, this also must be mentioned in the changelog. And it can be squashed with the commit introducing `--force`.